clients/fides-js/src/services/external/geolocation.ts

```diff
@@ -31,6 +32,11 @@ export const getGeolocation = async (
   const fetchOptions: RequestInit = {
     mode: "cors",
   };
+  if (forwarded_ip) {
+    const headers: Headers = new Headers();
+    headers.set("cloudfront-viewer-address", forwarded_ip);
```

Comment:
You're right about this not working with our existing lambda API. I also tried `x-forwarded-for` which didn't seem to work either. We may have to add a query param (or new custom header) to our lambda API instead of relying on any existing headers to forward IP address. What do you think?

Reply:
Due to the way that API functions, I think we're just not going to be able to get it to work - it's incredibly fast and performant _because_ it leverages edge CDNs, so it doesn't really have the capability to perform a slower, more traditional geoIP lookup.

We can instead use ipinfo.io or similar for this. I'd recommend taking this part of the code out for now, and shipping the enhanced prefetch for experience data in the _special case_ where the geolocation is specified up front (e.g. `?geolocation=us_ca`) and we can handle the IP lookup upstream in the infra